src/main/python/hydra/zmqtest/zmq_config.py

```diff
@@ -0,0 +1,6 @@
+# PUB server
+msg_count=10000  # Total messages to send
+msg_delay=0.001   # Delay between each message
+
+# SUB client
+total_apps = 200  # Total client apps to launch
```

Comment:
All of these should be run time arguments, as they will be used in many runs with different parameters.